scripts/check_no_trailing_whitespaces_or_tabs.sh

```diff
@@ -8,5 +8,5 @@ grep \
   --exclude-dir=data \
   --regexp '[[:blank:]]$' \
   --regexp $'\t' \
-  --line-number $(git ls-files)
+  --line-number $(git ls-files | grep -v -f .clang-format-ignore)
```

Comment:
Why so?  Does SWIG generate files with trailing whitespaces???

Reply:
I checked and current SWIG/Fortran uses tabs for enums.